src/dotnet/APIView/APIViewWeb/Client/src/helpers.ts

```diff
@@ -2,11 +2,10 @@
 export function updatePageSettings(callBack) {
   var hideLineNumbers = $("#hide-line-numbers").prop("checked");
   var hideLeftNavigation = $("#hide-left-navigation").prop("checked");
-  var selectedTheme = $("#theme-selector").children(":selected").val() as string;
-  var uri = location.origin + `/account/updatesettings?hideLineNumbers=${hideLineNumbers}&hideLeftNavigation=${hideLeftNavigation}&theme=${selectedTheme}`;
+  var uri = location.origin + `/account/updatereviewpagesettings?hideLineNumbers=${hideLineNumbers}&hideLeftNavigation=${hideLeftNavigation}`;
```

Comment:
shouldn't this API path be `/userprofile/updatereviewpagesettings`

Reply:
You are right. What's weird is that it still works with this in my local testing.